### PR TITLE
Add 'kleiderkreisel.de' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22kleiderkreisel-de%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22addresses%22%5D%2C%22name%22%3A%22kleiderkreisel.de%22%2C%22runs%22%3A%5B%22mamikreisel.de%22%5D%2C%22address%22%3A%22Wallstra%C3%9Fe%2065%5Cn10179%20Berlin%22%2C%22email%22%3A%22legal%40kleiderkreisel.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.kleiderkreisel.de%2Fimpressum%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F801%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

**Comment by mal-tee** [comment]
Thanks!

File: companies/kleiderkreisel-de.json [hunk 1]
```diff
@@ -1,20 +1,26 @@
 {
     "slug": "kleiderkreisel-de",
     "relevant-countries": [
-        "de"
+        "de",
+        "at"
     ],
     "categories": [
         "addresses"
```
Comment (mal-tee) [inline companies/kleiderkreisel-de.json:8]: note to self: fix this

**Comment by mal-tee** [comment]
@baltpeter Should we add https://www.vinted.com/ as well? according to the PP on they are the only site that is run by a US company.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**